tests/adapter/dbt/tests/adapter/constraints/test_constraints.py

```diff
@@ -190,17 +178,25 @@ def test__constraints_correct_column_data_types(self, project, data_types):
 create table {0} (
     id integer not null primary key check (id > 0) ,
     color text ,
-    date_day date
+    date_day text
 ) ;
 insert into {0} (
     id ,
     color ,
     date_day
-) (
+)
+(
```

Comment:
@gshank Is this the nested sql that's inexplicably failing? I recall you mentioning one, but can't remember which. I wanted to help look into it.

Reply:
I don't think anything is inexplicably failing right now. We were having a problem on dbt-spark because we hadn't specified file_format: delta, and the constraints were failing because they were referencing the original column name (not the spoofed one). But it's working now.